lib/Mojo/Util.pm

```diff
@@ -7,6 +7,7 @@ use Digest::MD5 qw(md5 md5_hex);
 use Digest::SHA qw(hmac_sha1_hex sha1 sha1_hex);
 use Encode 'find_encoding';
 use Exporter 'import';
+use File::Basename;
```

Comment:
I'm almost certain there are default exports polluting the namespace now.

Reply:
You can use Test::CleanNamespaces in an xt/author test to know for sure.

Reply:
Yes File::Basename has default exports, can be suppressed with `use File::Basename ()`